### PR TITLE
Remove messages which have not been manually checked

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

File: data_tools/firebase_client_wrapper.py [hunk 1]
```diff
@@ -189,6 +189,41 @@ def _delete_collection(coll_ref, batch_size):
...
+        # Get the latest label from each scheme
+        latest_labels = dict()  # of scheme id -> label
+        for label in msg["Labels"]:
+            if label["SchemeID"] not in latest_labels:
+                latest_labels[label["SchemeID"]] = label
+
+        manually_coded = False
+        for label in latest_labels.values():
+            if label["Checked"]:
+                manually_coded = True
```
Comment (lukechurch) [inline data_tools/firebase_client_wrapper.py]: break

File: data_tools/firebase_client_wrapper.py [hunk 2]
```diff
@@ -189,6 +189,41 @@ def _delete_collection(coll_ref, batch_size):
...
+
+        if not manually_coded:
+            print(f"Deleting message with sequence number: {msg['SequenceNumber']}")
+            client.document(f"datasets/{segment_id}/messages/{msg['MessageID']}").delete()
```
Comment (lukechurch) [inline data_tools/firebase_client_wrapper.py]: Could we count the number we're deleting to report at the end of the run?

File: data_tools/firebase_client_wrapper.py (hunk 1 above)
Comment (as2388) [reply data_tools/firebase_client_wrapper.py]: added

File: data_tools/firebase_client_wrapper.py (hunk 2 above)
Comment (as2388) [reply data_tools/firebase_client_wrapper.py]: added

**Pull request merged**